src/zocket/Program.cs

```diff
@@ -68,6 +82,95 @@ private static void Listen(int port, string command)
             exitEvent.WaitOne();
 
             process.Terminate();
+            return Task.CompletedTask;
+        }
+
+        private static async Task ListenWindows(int port, string command)
+        {
+            using var cancellationTokenSource = new CancellationTokenSource();
+            Console.CancelKeyPress += delegate (object sender, ConsoleCancelEventArgs e)
+            {
+                e.Cancel = true;
+                cancellationTokenSource.Cancel();
+            };
+
+            var parsedCommand = command.Split(' ', 2);
+            ProcessStartInfo psi = parsedCommand.Length switch
+            {
+                1 => new ProcessStartInfo(parsedCommand[0]),
+                2 => new ProcessStartInfo(parsedCommand[0], parsedCommand[1]),
+                _ => throw new ArgumentException(),
+            };
+
+            var currentAssembly = Assembly.GetExecutingAssembly().Location;
+            var reloadIntegrationPath = Path.GetFullPath(Path.Combine(currentAssembly, "..", "ReloadIntegration.dll"));
+
+            using var listenSocket = SetupSocket(port);
+
+            psi.EnvironmentVariables["DOTNET_STARTUP_HOOKS"] = reloadIntegrationPath;
+            psi.EnvironmentVariables["ASPNETCORE_HOSTINGSTARTUPASSEMBLIES"] = "ReloadIntegration";

```

Comment:
Move the shared stuff into a helper and share across both?

Reply:
Also append instead of nuking the variable

Reply:
I'd prefer to move stuff into shared after we stabilize the windows stuff. I can imagine this code path being merged together, removing the need for the helper.